components/custom_board/adau1961/adau1961.c

```diff
@@ -441,49 +441,8 @@ esp_err_t adau1961_init(audio_hal_codec_config_t *codec_cfg) {
 }
 
 esp_err_t adau1961_get_volume(int *vol) {
-  esp_err_t ret = ESP_OK;
-  uint8_t data[3];
-  int8_t volIndB;
-
-  // get current register setting of left channel
-  data[0] = (uint8_t)(R29_PLAYBACK_HP_LEFT_VOL_CTRL >> 8);
-  data[1] = (uint8_t)R29_PLAYBACK_HP_LEFT_VOL_CTRL;
-  ret =
-      i2c_bus_read_bytes(i2c_handler, adau1961_addr, &data[0], 2, &data[2], 1);
-  ADAU1961_ASSERT(ret, "Fail to read R29_PLAYBACK_HP_LEFT_VOL_CTRL", ESP_FAIL);
-
-  //  volIndB = ((data[2] >> 2) & 0x3F) - 57;
-  //*vol = pow(10, ((float)volIndB / 20.0)) * 50.0;
-
-  volIndB = ((data[2] >> 2) & 0x3F);
-  *vol = (int)volIndB * 100 / 63;
-
-  // we assume all channel are set to the same volume.
-  // this is a save assumption, as we are setting them in
-  // adau1961_set_volume()
+  
```

Comment:
don't change this file please

Reply:
Could you restore it? I got some problems with VSCODE and I edited the wrong file 😅

Reply:
please do a git checkout for this file to restore it